Dockerfile

```diff
@@ -3,15 +3,15 @@ FROM node:19.8.1-alpine as builder
 COPY . /frontend-shared
 RUN cd /frontend-shared && \
     yarn install --frozen-lockfile && \
-    yarn build-pattern-lib
+    NODE_ENV=production yarn build-pattern-lib
```

Comment:
We cannot set `ENV NODE_ENV=production` for the whole docker stage, because then `yarn install` does not install dev dependencies, which are needed for bundling.